.github/workflows/static.yml

```diff
@@ -29,6 +29,32 @@ jobs:
           make get-deps
           make static-check
 
+  init-check:
```

Comment:
From the description:

> All integration changes are additive so they will not mutate or interrupt our existing processes.

So the plan for now is to be sure that all the changes won't have any effect on the current build process -- this is why all the init-specific targets exist rather than adding them directly to the existing targets.